source/draw/gpu/opengl/backend/gles3/Gles3Debug.ooc

```diff
@@ -41,6 +41,12 @@ validate: func (message: String) {
 		Debug print(errorMessage)
 		raise(errorMessage)
 	}
+	eglError := eglGetError()
+	if (eglError != EGL_SUCCESS) {
+		errorMessage := "EGL error " + message + ": " + getEglErrorMessage(eglError)
+		Debug print(errorMessage)
+		raise(errorMessage)
```

Comment:
@emilwestergren `Debug error` calls `raise`, if that's what you mean?

@erikhagglund It works, but is not equivalent, since it will not print the error message through the specified debug output channel.